config/routes.rb

```
@@ -1,217 +1,237 @@
+      resources :questionnaire_types, only: [:index] do
+      end
+
+      resources :item_types, only: [:index] do
+      end
+
+
+      resources :questions do
+        collection do
+          get :types
+          get :quiz_types
+          get 'show_all/questionnaire/:id', to:'questions#show_all#questionnaire', as: 'show_all'
```

RuboCop (1.86.1)

[convention] Space missing after colon.

(Layout/SpaceAfterColon)